suite-common/graph/src/graphDataFetching.ts

```diff
@@ -153,6 +157,12 @@ const getAccountBalanceHistory = async ({
     }
 
     const getBalanceHistory = async () => {
+        if (isIgnoredBalanceHistoryCoin(coin)) {
```

Comment:
Is this needed? I believe that the networkSymbol of a graph-disabled coin should not me passed to the graph calculation at all? What do you think about filtering the coins before passing them to the graph calculation? Then this change nor the change in `graphBalanceEvents` would not be needed.